File: app/src/main/java/com/example/capstona_a/data/Ingame.java
package com.example.capstona_a.data;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;


@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Ingame {

    @SerializedName("status")
    @Expose
    private Integer status;
    @SerializedName("gameMode")
    @Expose
    private String gameMode;


    @SerializedName("gameType")
    @Expose
    private String gameType;
    @SerializedName("players")
    @Expose
    private List<Player> players;


}

File: app/src/main/java/com/example/capstona_a/data/Ingame.kt
package com.example.capstona_a.data

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


data class Ingame (
    @SerializedName("status")
    @Expose
    private val status: Int? = null,

    @SerializedName("gameMode")
    @Expose
    private val gameMode: String? = null,

    @SerializedName("gameType")
    @Expose
    private val gameType: String? = null,

    @SerializedName("players")
    @Expose
    private val players: List<Player>? = null,
    @SerializedName("predict")
    @Expose
    private val predict: Double? = null,

)
{
    override fun toString(): String {
        return "Ingame(predict=$predict)"
    }
    fun getPlayer(): List<Player>?{
        return this.players;
    }
}
